lua/svartafanan/timer.lua

```diff
@@ -0,0 +1,41 @@
+local M = {}
```

Comment:
Good stuff.

Could I ask you to add a few comments?
As you see I try to comment a lot as I find it makes it easier to come back to the code again. Even if it's obvious right now it might not be when I come back.

Reply:
Ofc, I'll add comments.